packages/super-components/components/msc-tail-logs/mock/v6/me-contact.json

```diff
@@ -0,0 +1,14 @@
+{
+  "address": {
+    "country": "FR",
+    "city": "Roubaix",
+    "zip": "59100",
+    "line1": "La rue du Test"
+  },
+  "firstName": "Annie Galou",
+  "id": 11111111,
+  "language": "fr_FR",
+  "lastName": "Lalie Panouc",
+  "legalForm": "individual",
+  "phone": "+33.123456789"
+}
```

Comment:
**question:** Why a generic component like `msc-tail-logs` has to depend on User info?

Reply:
It does not : I just forgot to remove the mock. It is included in the example component in the super-component generator.